### ISSUE TITLE
Setting category of a game causes Steam to crash

### ISSUE THREAD

**Issue by Shished** [ValveSoftware/steam-for-linux#4073]
[PC and OS specs from Steam](http://pastebin.com/fjpJu5LR)
Steam always crashes when trying to assign game to a category or to hide game from library.
Crash log:
```
crash_20151018192049_1.dmp[12423]: Uploading dump (out-of-process)
/tmp/dumps/crash_20151018192049_1.dmp
~/.local/share/Steam/steam.sh: line 756: 12280 Segmentation fault      (core dumped) $STEAM_DEBUGGER "$STEAMROOT/$STEAMEXEPATH" "$@"
[1018/192105:WARNING:pref_notifier_impl.cc(27)] pref observer found at shutdown plugins.always_authorize
[1018/192105:WARNING:pref_notifier_impl.cc(27)] pref observer found at shutdown plugins.allow_outdated
Refresh rate: 1
Refresh rate: 1
Refresh rate: 1
Refresh rate: 1
Refresh rate: 1
Refresh rate: 1
Refresh rate: 1
Refresh rate: 1
crash_20151018192049_1.dmp[12423]: Finished uploading minidump (out-of-process): success = yes
crash_20151018192049_1.dmp[12423]: response: CrashID=bp-7ef50b39-cbb3-43b7-9311-6575c2151018
crash_20151018192049_1.dmp[12423]: file ''/tmp/dumps/crash_20151018192049_1.dmp'', upload yes: ''CrashID=bp-7ef50b39-cbb3-43b7-9311-6575c2151018''
```
This problem appears in both stable and beta versions of Steam.

**Comment by Footissimo** [comment]
[Same for me](http://pastebin.com/kSkvtCy4) on both stable and beta client.

[System Specs](http://pastebin.com/yBR2bxr3)

**Comment by rinaldus** [comment]
I had this bug several days ago. It happened when I changed from stable nvidia-drivers 355.11 to unstable 358.13. When I noticed this bug, I downgraded them back to stable and this bug dissappeared.
My OS is Gentoo.

**Comment by BlaXpirit** [comment]
I have this too. As a matter of fact, I did recently upgrade to NVIDIA drivers 358.16.

[Computer specs](http://blaxpirit.com/specs.html)

**Comment by BlaXpirit** [comment]
Downgraded my packages to the state they were in yesterday:
```
libpaper-1.1.24 lapack-3.5.0 cblas-3.5.0 breeze-icons-5.16.0 blas-3.5.0 virtualbox-5.0.8 virtualbox-host-modules-5.0.8 virtualbox-host-dkms-5.0.8 nvidia-355.11 lib32-nvidia-libgl-355.11 lib32-nvidia-utils-355.11 nvidia-utils-355.11 xorg-server-1.17.4 nvidia-libgl-355.11 xf86-input-evdev-2.10.0 xorg-server-common-1.17.4
```

Categories work well again.

**Comment by Swiftpaw** [comment]
Downgrading NVIDIA drivers back to 355 fixes this bug.

Is this a NVIDIA or Steam bug?

**Comment by LeoVerto** [comment]
This issue seems to be related to https://github.com/ValveSoftware/steam-for-linux/issues/4144

**Comment by scrawl** [comment]
Happened to me with the stable steam client and nvidia drivers 358.16.

<pre>
Processor Information:
    Vendor:  GenuineIntel
    CPU Family:  0x6
    CPU Model:  0x3c
    CPU Stepping:  0x3
    CPU Type:  0x0
    Speed:  4600 Mhz
    8 logical processors
    4 physical processors
    HyperThreading:  Supported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported
    
Network Information:
    Network Speed:  
    
Operating System Version:
    Ubuntu 14.04.3 LTS (64 bit)
    Kernel Name:  Linux
    Kernel Version:  3.16.0-59-generic
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11600000
    X Window Manager:  Compiz
    Steam Runtime Version:  steam-runtime-release_2015-06-12
    
Video Card:
    Driver:  NVIDIA Corporation GeForce GTX 980/PCIe/SSE2

    Driver Version:  4.5.0 NVIDIA 358.16
    OpenGL Version: 4.5
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 119 Hz
    VendorID:  0x10de
    DeviceID:  0x13c0
    Number of Monitors:  1
    Number of Logical Video Cards:  1
    Primary Display Resolution:  2560 x 1440
    Desktop Resolution: 2560 x 1440
    Primary Display Size: 23,54" x 13,23"  (26,97" diag)
                                            59,8cm x 33,6cm  (68,5cm diag)
    Primary Bus: PCI Express 16x
    Primary VRAM: 4096 MB
    Supported MSAA Modes:  2x 4x 8x 16x 
    
Sound card:
    Audio device: Realtek ALC892
    
Memory:
    RAM:  15990 Mb
    
Miscellaneous:
    UI Language:  English
    LANG:  en_US.UTF-8
    Microphone:  Not set
    Total Hard Disk Space Available:  1488876 Mb
    Largest Free Hard Disk Block:  647195 Mb
    
Installed software:
    
Recent Failure Reports:
    Sat Jan 30 00:10:19 2016 GMT: file ''/tmp/dumps/crash_20160130002247_1.dmp'', upload yes: ''CrashID=bp-9ac49a60-3b5c-4922-870f-9bca82160129''
    Sat Jan 30 00:10:38 2016 GMT: file ''/tmp/dumps/crash_20160130011013_1.dmp'', upload yes: ''CrashID=bp-472b5ff6-44ae-4f06-aa07-fb2872160129''
    
</pre>

**Comment by Tele42** [comment]
@scrawl there is a known issue with nvidia 358, use another version.
